### PR TITLE
How it works

### PR DESCRIPTION
**Pull request by hicondron**
How it works html and css rough out complete

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -9,4 +9,28 @@ html {
 
 body {
     font-family: 'Comfortaa', cursive;
+}
```
Comment (hicondron) [inline css/style.css:12]: added basic styling for how it works section

File: index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" href="/css/style.css"> 
+    <link rel="stylesheet" type="text/css" href="css/style.css">
```
Comment (hicondron) [inline index.html:7]: changed src to href for chrome to recognize stylesheet

File: index.html [hunk 3]
```diff
@@ -16,7 +16,42 @@
 
         <!-- HOW IT WORKS SECTION -->
         <section id="how_to"></section>
-
+            <div class= "title">
```
Comment (hicondron) [inline index.html:19]: add how it works section

**Review by hicondron — COMMENTED** [review]
change src to href for stylesheet to work
add basic css to section

**Review by BachAmadou — APPROVED** [review]

**Pull request merged**